### PR TITLE
feat: Not analyzed data color

### PR DESCRIPTION
**Pull request by ahmedfarouk2000**
Consistant color for regions with no data, and update legends with not analyzed data point.

### CODE REVIEW

File: src/styles/globals.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
 @tailwind components;
 @tailwind utilities;
 
+:root {
+  --nextui-notAnalyzed: 0, 0%, 55%;
```
Comment (bohdangarchu) [inline src/styles/globals.css:6]: do we need this if it is already defined in tailwind config?

**Review by bohdangarchu — APPROVED** [review]
lgtm!

**Review by marinovl7 — CHANGES_REQUESTED** [review]
Great job, however this is somehow not working for the IPC Country View. Can you have a look? 

image

File: src/styles/globals.css (hunk 1 above)
Comment (ahmedfarouk2000) [reply src/styles/globals.css]: the color was kinda transparent, but anyways I have applied another color and removed this part.

**Comment by ahmedfarouk2000** [comment]
> Great job, however this is somehow not working for the IPC Country View. Can you have a look?
> 
> image

done

**Pull request merged**